Library/Homebrew/os/linux/elf.rb

```diff
@@ -198,13 +208,54 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def rpath_using_patchelf_rb
+    patcher = patchelf_patcher
+    patcher.runpath || patcher.rpath
+  rescue PatchELF::PatchError => e
+    opoo e
```

Comment:
I'm thinking this ought to be a hard error rather than a warning. If so, this `rescue` can be removed.

Reply:
`odie` perhaps ?

Reply:
I believe that's the default behaviour, that an uncaught exception will `odie`, so there's no need for the `rescue` block in that case. Please test! You can use a super fake test that just does `raise PatchELF::PatchError, "an error"`